code/modules/organs/external/_external.dm

```diff
@@ -58,7 +58,7 @@
 	// Joint/state stuff.
 	var/joint = "joint"                // Descriptive string used in dislocation.
 	var/amputation_point               // Descriptive string used in amputation.
-	var/dislocated = 0                 // If you target a joint, you can dislocate the limb, causing temporary damage to the organ.
+	var/dislocated = 0                 // If you target a joint, you can dislocate the limb, causing temporary damage to the organ. Boolean.
```

Comment:
Could use a `limb_status` flag for this considering it's no longer a trinary condition.

Reply:
That's a good idea! 
Speaking of limb status, some of the state in there probably shouldn't be a status, like prosthetic and assisted. Because those are more like permanent changes, and also because rejuv resets them on rejuv cause they happen to be set in the status flag.

Reply:
To be clearer, I feel like ORGAN_ASSISTED, ORGAN_PROSTHETIC, and ORGAN_CRYSTAL probably should be limb flags and not status flags. Since they define permanent capabilities and not transient states? I could probably do something on another PR if you feel it makes sense.

Reply:
I don't disagree but I also don't personally think it's a huge priority currently.